packages/sidebar-settings/src/blocks/legacyTemplateInput.ts

```diff
@@ -0,0 +1,15 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import type { BaseBlock } from './base';
+
+export type LegacyTemplateInputBlock<AppBridge> = {
+    /**
+     * The setting type.
+     */
+    type: 'legacyTemplateInput';
+
+    /**
+     * Whether multiple templates can be selected.
+     */
+    multiSelection?: boolean;
```

Comment:
`multiSelection` is also defined in `TemplateInputBlock` type but never used

Reply:
I've removed `multiSelection` for now for both blocks. This prop came from an asset input block which I used as an example for a template block.